lib/pbench/server/indexing_tarballs.py

```diff
@@ -267,6 +272,32 @@ def process_tb(self, tarballs):
         else:
             idxctx.set_tracking_id(tracking_id)
 
+        return res.value
+
+    def process_tb(self, tarballs: List[TarballData]) -> int:
+        """Process Tarballs For Indexing and creates report
+
+        Args:
+            tarballs:   List of tarball information tuples
+
+        Returns:
+            status code
+        """
+        res = 0
```

Comment:
I believe that this initialization is pointless, since it is unconditionally overwritten at line 293.  (I'm surprised your IDE didn't mention this to you....)